dev-php/pecl-mcrypt/Manifest

```diff
@@ -0,0 +1 @@
+DIST mcrypt-1.0.1.tgz 33782 BLAKE2B db5f89fdbc64c81bd50c92301155527f40b28cd274312565c79327133baa20e14c488f8ad1f87e6ddfd3ced9706ca30f10dc69a08279c82c9ec83fb5f713e826 SHA512 1677202715499a41ac6999089257a059106e58cbac06ca8a7dd2e9edd28be5a4118234dd7468110e9e3d7f9de5d099367848fe84bcd06716ea56645c9015c578
```

Comment:
Seems like it's a general problem (also checked pecl-redis for example). But since we don't expect much releases of mcrypt/pecl-mcrypt, I think we can live with this compromise for now.